src/Dofs/ConstraintHandler.jl

```diff
@@ -588,12 +588,18 @@ function apply!(KK::Union{SparseMatrixCSC,Symmetric}, f::AbstractVector, ch::Con
 end
 
 # Similar to Ferrite._condense!(K, ch), but only add the non-zero entries to K (that arises from the condensation process)
-function _condense_sparsity_pattern!(K::SparseMatrixCSC, acs::Vector{AffineConstraint})
+function _condense_sparsity_pattern!(K::SparseMatrixCSC{T}, acs::Vector{AffineConstraint{T}}) where T
     ndofs = size(K, 1)
-
+    (length(acs) == 0) && return 
     # Store linear constraint index for each constrained dof
     distribute = Dict{Int,Int}(acs[c].constrained_dof => c for c in 1:length(acs))
 
+    #Adding new entries to K was extremely slow, so craete a new sparsity triplet for the condensed sparsity pattern
+    N = length(acs)*10 # TODO: Better size estimate for additional condensed sparsity pattern.
```

Comment:
Perhaps just
```suggestion
    N = length(acs) # TODO: Better size estimate for additional condensed sparsity pattern.
```
? It grows by a factor 1.5 later so that isn't so many resizes I think.